config/purposes/scrna_core_library_prep.yml

```diff
@@ -116,8 +116,9 @@ LRC GEM-X 5p CITE SUP:
   :input_plate: false
 LRC GEM-X 5p CITE SUP XP:
   :asset_type: plate
-  :stock_plate: false
-  :input_plate: false
+  :stock_plate: true # Uploaded via sample manifest.
+  :input_plate: true # Passed when submission is built.
```

Comment:
Should we close the request `limber_scrna_core_cdna_pcr_cite` first for this work because the submission will be adding the new request `limber_scrna_core_library_prep`?